spruce/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM golang:1.5.1-alpine
 
-RUN apk update
-RUN apk add git \
+ENV SPRUCE_VERSION 0.12.0
+
+RUN apk add --update git \
   && go get github.com/geofffranks/spruce \
```

Comment:
If you use `go get -d` here it'll avoid compiling at this stage (`-d` means download only).